kernel/kernel-api/src/main/java/io/delta/kernel/internal/replay/LogReplay.java

```diff
@@ -351,6 +357,27 @@ private Optional<Long> loadLatestTransactionVersion(Engine engine, String applic
     return Optional.empty();
   }
 
+  /**
+   * Loads the domain metadata map, either from CRC info (if available) or from the transaction log.
+   * Note that when loading from CRC info, tombstones (removed domains) are not preserved, while
+   * they are preserved when loading from the transaction log.
+   *
```

Comment:
Do you know if there's any reason we would want the tombstones? I think if they are never used we should just standardize this to only return the active domains

Reply:
I just keep the old behavior to limit the change of this PR.

>I think if they are never used we should just standardize this to only return the active domains
Let me file a tracking issue